scripts/globals/spell_data.lua

```diff
@@ -639,8 +639,8 @@ xi.magic.spell =
     DISPELGA              =  360,
     BLINDGA               =  361,
     BINDGA                =  362,
-    SLEEPGA               =  363,
```

Comment:
Could you leave the names there and just comment out the lines?

Reply:
![image](https://user-images.githubusercontent.com/1389729/123920873-18874100-d98f-11eb-8634-754a52d8b7f0.png)

![image](https://user-images.githubusercontent.com/1389729/123920896-20df7c00-d98f-11eb-9e0a-aa12d06d2928.png)

They are duplicated in the DATs too... what should we do with this?

Reply:
Since we only have defined the first set in the database, names could be kept, commented out, with a note like...
"Duplicated spell entry in the DATs"